examples/skip_plot_brighest_pixel.py

```diff
@@ -0,0 +1,54 @@
+"""
+=======================================
```

Comment:
Was this meant to be a new example?

Reply:
Yeah, I thought this would be better as a separate example

Reply:
I don't think there is enough different here to warrant a new example?